### DISCUSSION TITLE
Using Azure DevOps Extensibility in VS 2022

### DISCUSSION THREAD

**Discussion by sharpninja** [VsixCommunity/Community.VisualStudio.Toolkit#485] [General]
I am writing an Azure DevOps extension for my company to help us with workflow using ADO.  My original work was in a WPF app, but I am migrating it to a VS Extension.  The current code authenticates the user with ADO each time the Extension starts by popping the dialog from the nuget package.  I would like to be able to use the logged in user to VS instead.  How would I go about this?

**Comment by StevenRasmussen** [comment]
I could be wrong but I don't believe there's any way to take advantage of the logged in user in VS.  You "might" be able to get the username somehow but that would be the extent of it... if that's even possible.
